crates/typst-library/src/text/mod.rs

```diff
@@ -215,6 +215,18 @@ pub struct TextElem {
     #[default(Rel::one())]
     pub spacing: Rel<Length>,
 
+    /// Whether to automatically insert spacing between CJK and Latin characters.
+    ///
+    /// ```example
+    /// #set text(cjk-latin-spacing: true)
+    /// 第4章介绍了基本的API。
+    ///
+    /// #set text(cjk-latin-spacing: false)
+    /// 第4章介绍了基本的API。
+    /// ```
+    #[default(true)]
+    pub cjk_latin_spacing: bool,
```

Comment:
If we intend on adding more customization to this in the future, then perhaps this should use an `auto` / `none` toggle instead to avoid unnecessary breaking changes in the future.

Not sure of the best way to represent this though (I was thinking of something like `Option<Smart<()>>`, but that might not work). Perhaps `Option<AutoTypeOrWhateverItsNameIs>`? Or `Smart<NoneType>` or something similar

Reply:
Make sense. Changes made.